interfaces/builtin/optical_drive.go

```diff
@@ -35,13 +35,22 @@ const opticalDriveConnectedPlugAppArmor = `
 @{PROC}/sys/dev/cdrom/info r,
 `
 
+const opticalDriveConnectedPlugUdev = `
+# This file contains udev rules for optical drives.
+#
+# Do not edit this file, it will be overwritten on updates
+
+KERNEL=="sr[0-9]*", NAME="%%k", SYMLINK+="scd%%n", TAG+="%s"
```

Comment:
We can drop NAME and SYMLINK. /dev/scd* aren't always symlinks, so adjust to be:
```
KERNEL=="sr[0-9]*",  TAG+="%[1]s"
KERNEL=="scd[0-9]*", TAG+="%[1]s"
```